samples/oauth_rails_vetverification/app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,60 @@
+class AuthController < ApplicationController
+  def login
+    nonce_base = SecureRandom.base64(20)
+    session[:nonce_key] = nonce_base
+    session[:login_time] = Time.zone.now.to_i
+    scope = 'openid profile service_history.read disability_rating.read veteran_status.read'
+    oauth_params = {
+      client_id: ENV['va_developer_client_id'], # required - The client_id issued by the VA API Platform team
+      nonce: digest(nonce_base), # optional - Used with id_token to verify token integrity. Ensure the nonce in your id_token is the same as this value.
+      redirect_uri: 'http://localhost:3000/callback', # required - the URL you supplied that the Veteran will be redirected to after authorizing your application
+      # response_mode: 'fragment', # optional - Either fragment or query, recommended not to use unless you have a specific reason. Defaults to fragment.
+      response_type: 'code', # required - one or two of, id_token, token, or code. Using code will require your application to complete the Authorization Code Flow. Using id_token or token allows you to use the Implicit flow.
+      scope: scope, # optional - Will use your application's default scopes unless you specify a smaller subset of scopes separated by a space.
```

Comment:
Might be worth mentioning that if you do specify scopes explicitly, those scopes *must* include `openid`

Reply:
yeah, those comments were c&p from the doc when I was building this out.  But my experience was my oauth was rejected if I didn't have a scope param.
I pulled out those comments and just put a link to the docs.